test/model/r_forecast/test_r_hierarchical_predictor.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+
+from gluonts.core import serde
+from gluonts.dataset.util import forecast_start
+from gluonts.evaluation import MultivariateEvaluator, backtest_metrics
+from gluonts.model.forecast import SampleForecast
+from gluonts.model.r_forecast import (
+    RHierarchicalForecastPredictor,
+    R_IS_INSTALLED,
+    RPY2_IS_INSTALLED,
+    SUPPORTED_HIERARCHICAL_METHODS,
+)
+
+
+# conditionally skip these tests if `R` and `rpy2` are not installed
+if not R_IS_INSTALLED or not RPY2_IS_INSTALLED:
+    skip_message = "Skipping test because `R` and `rpy2` are not installed!"
+    pytest.skip(msg=skip_message, allow_module_level=True)
+
+from test.model.deepvar_hierarchical.generate_hierarchical_dataset import sine7
```

Comment:
Running this locally I get the following:
`ModuleNotFoundError: No module named 'test.model'`

We need to relocate the dataset sine7 so that it can be used in this test and the test in `test/model/deepvar_hierarchical/test_deepvar_hierarchical.py`.

A possibility for this would be to relocate sine7 as a fixture in a `conftest.py` so that sine7 can be passed to both tests.